src/Types/Attribute/Collection.php

```diff
@@ -102,17 +102,17 @@ public function offsetExists($offset)
         return true;
     }
 
-    public function offsetGet($offset)
+    public function offsetGet($offset): array|bool|callable|int|float|null|object|string
```

Comment:
AKA `mixed` ?

Reply:
Yep was a cut and paste from source, changed